src/modules/Users.pm

```diff
@@ -3979,7 +3829,6 @@ sub WriteLoginDefaults {
 
 ##------------------------------------
 # Save Security settings (encryption method) if changed in Users module
-BEGIN { $TYPEINFO{WriteSecurity} = ["function", "boolean"]; }
```

Comment:
I'd keep the type data in a comment, it might be useful later when reimplementing in Ruby. Maybe you could use the yardoc format now so the refactoring it will be easier in the future...